cargo_config.in

```diff
@@ -0,0 +1,3 @@
+[target.@TARGET@]
```

Comment:
If you've looked at the alternative and this is the best option, then I'm happy to go with it.